packages/java/tests/spring/endpoints-discovery/pom.xml

```diff
@@ -70,6 +70,18 @@
             <plugin>
                 <groupId>com.vaadin</groupId>
                 <artifactId>flow-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>prepare-frontend</goal>
+                            <!--
+                             Needs explicit frontend build, because the tests
+                             dont wait for DevServer's startup frontend tasks
+                              -->
+                            <goal>build-frontend</goal>
+                        </goals>
+                    </execution>
+                </executions>
```

Comment:
If so, it is probably better to remove this here and restore it at line 149 in `packages/java/tests/pom.xml`. I removed it in this PR, but it is maybe needed everywhere.

Reply:
I’ll keep both, as they are enabled for different reasons. In this case its test assertions, whereas everywhere else it is for CI performance.

Reply:
Ah, you cleaned it up. Thanks!